nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster/src/test/java/org/apache/nifi/cluster/integration/Node.java

```diff
@@ -311,6 +320,11 @@ public boolean hasRole(final String roleName) {
         return leaderAddress.equals(getClusterAddress());
     }
 
+    public void addProcessor(final Processor example) throws ProcessorInstantiationException {
```

Comment:
You're right - it shouldn't be there. I started to add some additional methods to improve the power of the testing by allowing it to change the flow so that we could easily test how the cluster behaves when the flows differ between nodes. But then I removed the methods because they didn't belong on this PR and it looks like I forgot to remove that method.